src/main/java/ch/njol/skript/SkriptCommandTabCompleter.java

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
Hitting tab on `/sk update` gives `/sk changes`
Hitting tab on `/sk reload` gives `/sk aliases`
Hitting tab on `/sk enable` gives `/sk %disabled script%`
Hitting tab on `/sk disable` gives `/sk all`

This is the same as https://github.com/SkriptLang/Skript/pull/3916#discussion_r742749694, make sure you actually fix review comments before marking as resolved